packages/eslint-config-4catalyzer/rules.js

```diff
@@ -80,4 +86,13 @@ module.exports = {
     },
   ],
   'import/prefer-default-export': 'off',
+  'import/order': [
```

Comment:
do we want case-insensitivity in the ordering? https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md#alphabetize-order-ascdescignore-caseinsensitive-truefalse